pages/How-to/guide/starting-discussion.md

```diff
@@ -1,4 +1,17 @@
-<div align="center">
-  <img src="https://user-images.githubusercontent.com/74750414/169188784-680014eb-ae5a-4062-b9c0-40276f28a261.png" />
-</div>
+## Starting Discussion in GitHub
+1. Go to the **repo** in which you want to start a discussion in or use this link - **https://github.com/{repo-owner}/{repo-name}/discussions**.
```

Comment:
```suggestion
1. Go to the **repo** in which you want to start a discussion in or use this link - `https://github.com/{repo-owner}/{repo-name}/discussions`.
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix